src/index.js

```diff
@@ -1,11 +1,11 @@
 import 'better-log/install';
 
-module.exports = function ({ Plugin, types: t }) {
-	return new Plugin('hello-world', {
+module.exports = function ({ types: t }) {
+	return {
 		visitor: {
-			Program(node, parent, scope, file) {
-				this.unshiftContainer('body', t.expressionStatement(t.literal('use helloworld')));
+			Program(path, file) {
+				path.unshiftContainer('body', t.expressionStatement(t.stringLiteral('use helloworld')));
```

Comment:
Not sure about it - it makes harder to convert literals between types (e.g. number <-> string) as you need to modify type as well apart from the value.

Anyway, discrepancies from the standard without a really good reason are always a sad thing IMO.